### DISCUSSION TITLE
Permissions get erased when saving own profile

### DISCUSSION THREAD

**Discussion by cfpinto** [thedevdojo/voyager#5395] [Q&A]
## Version information

 - Laravel: v8.49.1
 - Voyager: v1.4.x-dev
 - PHP: 7.4
 - Database: MySQL 5.7|8.0

## Description

When editing my own profile and click save my permissions are erased, both in the users table and the user_role table.

## Steps To Reproduce
1. Go to admin home
2. Click on your avatar on the top right corner
3. Click "Profile"
4. Click "Edit My Profile"
5. Click "Save"

- the system will enter redirect loop both with browser cache disabled and enabled
- user table will have empty role_id for logged in user
- user_role table will have no entries for logged in user

## Expected behavior
Permissions are expected to be preserved when saving user profile

## Additional context
Only happens for logged in user. when editing somebody else's profile works just fine

**Comment by MrCrayon** [comment]
Is this a new install or it started happening with an update?
Are you using custom Controllers/views?

**Comment by cfpinto** [comment]
Upgraded dependencies laravel 6 > 8 & voyager 1.1 (if i remember correctly) to 1.4.x-dev, not sure it was happening before users hadn't report it. No custom controllers or views.

thanks

**Comment by MrCrayon** [comment]
You probably forgot to set the controller in users BREAD settings.
Make sure you follow all upgrade guides 1.1 to 1.2, 1.2 to 1.3 and 1.3 to 1.4
https://voyager-docs.devdojo.com/v/1.2/getting-started/upgrading

I'm moving this to discussions.
